lib/src/base-styles/_nx-radio-checkbox.scss

```diff
@@ -6,6 +6,19 @@
  */
 @use '../scss-shared/nx-container-helpers';
 @use '../scss-shared/nx-text-helpers';
+@use '../scss-shared/nx-dark-mode-helpers';
+
+:root {
+  --nx-radio-checkbox-border-width: 1px;
```

Comment:
Why is this here and not in `_nx-variables.scss`?

Reply:
> Why is this here and not in `_nx-variables.scss`?

Totally forgot to move it there: 2353f6e1bcb99869300b48b54245e083c1f0d210